chirp/drivers/tk11.py

```diff
@@ -0,0 +1,3297 @@
+﻿# Copyright 2012 Dan Smith <dsmith@danplanet.com>
+# Copyright 2025 Thibaut Berg <thibaut.berg@hotmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import ctypes
+import enum
+import logging
+import struct
+import time
+import secrets
+
+from chirp import bitwise
+from chirp import chirp_common, memmap, directory, errors
+from chirp.bitwise import arrayDataElement
+from chirp.chirp_common import CloneModeRadio
+from chirp.settings import (RadioSetting, RadioSettingGroup,
+                            RadioSettings, RadioSettingValueList,
+                            RadioSettingValueString, RadioSettingValueBoolean,
+                            RadioSettingValueInteger, RadioSettingValueFloat,
+                            RadioSettingSubGroup)
+
+LOG = logging.getLogger(__name__)
+
+DEBUG = True
+DEBUG_SERIAL = True
+
+MEM_FORMAT = """
+struct {
+    u8 flag;
+} channels_usage[999];
+#printoffset "After channels usage";
```

Comment:
`#printoffset` is for debugging stuff, please don't leave them in code we expect to go into the tree as they just generate always-there noise in the log.